src/cli/mod.rs

```diff
@@ -1,5 +1,6 @@
 use anyhow::{Context, Result};
 use clap::{ArgAction, CommandFactory, Parser};
+use remove::RemoveSubcommand;
```

Comment:
I assume this was an auto import, so rust-analyzer put it up here, but all the imports for commands are grouped below so the new import should be there as well.